### PR TITLE
Upload test coverage status to code climate

### PR DESCRIPTION
**Pull request by Joyfal**
## Checklist

- [N/A] Did you write tests for your code?
- [N/A] Did all the tests pass locally?

## Trello Ticket
https://trello.com/c/PGGQdLDq/60-mvp-demo

## Short Description
Update code coverage status to code climate for badges and show

## Reviewer Notes

Anything special you want the reviewer to know?

### CODE REVIEW

File: .github/workflows/test_coverage.yml [hunk 1]
```diff
@@ -64,3 +64,9 @@ jobs:
           external_repository: UCSD-MASHED/TaterTalkCoverage
           publish_branch: main
           publish_dir: ./coverage
+      - name: Update code coverage to code climate
```
Comment (albinencio) [inline .github/workflows/test_coverage.yml:67]: Should be Upload?

Comment (Joyfal) [reply .github/workflows/test_coverage.yml:67]: I think it's fine.

**Review by albinencio — APPROVED** [review]

**Comment by macedo22** [comment]
@Joyfal I notice that the test overage indicator is blank (has a "?"). Is this a flaw in the setup, or is this expected?

**Comment by Joyfal** [comment]
> @Joyfal I notice that the test overage indicator is blank (has a "?"). Is this a flaw in the setup, or is this expected?

Once I merge it, it should show up after the github action is run.

**Pull request merged**